### PR TITLE
Removed duplicated svgs

### PR DESCRIPTION
**Pull request by Thomas-Boi**
This removes the duplicates listed in #607.

Notes:
- Spring, perl, and phoenix's plain versions were the same as their original but with the color stripped. The font versions also have the color stripped so we don't have to worry much about this.
- Lua actually don't have any duplicates, this has been crossed out in the issue
- Also centered `python-original.svg`
- Added `screenshots` folder to `.gitignore`. This is probably created due to one of the build steps

### CODE REVIEW

**Comment by Thomas-Boi** [comment]
Since `python-original.svg` was changed, this is also our first official test of the `svg-optimizer` bot. Let's see how it do.

**Comment by Thomas-Boi** [comment]
So it works great as expected. I'm also pleasantly surprised that the ids are prefixed properly when it's already prefixed.

**Comment by Thomas-Boi** [comment]
Just updated the font files and the `icomoon.json` to the proper names. Also tested it and everything looks fine in the Icomoon website.

**Comment by Thomas-Boi** [comment]
Just pushed a commit optimizing the `python-wordmark-original.svg` as seen in #103

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ node_modules
 geckodriver.log
 __pycache__
 *.pyc
-new_icons.png
\ No newline at end of file
+new_icons.png
+screenshots
```
Comment (Panquesito7) [inline .gitignore:8]: Missing an endline.
```suggestion
screenshots

```

**Review by amacado — COMMENTED** [review]
I support the idea of reducing the overall size by removing duplicates. But since this is a breaking change I would like to discuss if it's worth it. On one hand we have the reduced file size on the other hand it could break existing applications when someone (like me) is using the icons in an automated way. How much space do we save when removing those icons?

**Comment by Thomas-Boi** [comment]
> I support the idea of reducing the overall size by removing duplicates. But since this is a breaking change I would like to discuss if it's worth it. On one hand we have the reduced file size on the other hand it could break existing applications when someone (like me) is using the icons in an automated way. How much space do we save when removing those icons?

Not a lot actually. The files are between 1 kB to 4 kB so I'll use 2 kB per file. There are 12 files removed => `12 * 2 kB = 24 kB`. The benefit is small I admit and I have no strong feeling about the PR.

**Comment by Thomas-Boi** [comment]
@amacado I think you are right. The file savings are not a lot but the potential backwards compatibility issue might be worth more. I'm cool if you want to close this PR. I'll make another one for the other fixes.

**Pull request closed without merging**